File: app/src/main/java/cedule/app/data/entities/Category.java
package cedule.app.data.entities;

import android.graphics.Color;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "categories")
public class Category {
    @PrimaryKey
    public Integer id;

    @NonNull
    public String name;

    public Integer color;
}

File: app/src/main/java/cedule/app/data/entities/Category.kt
package cedule.app.data.entities;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "categories")
data class Category(
    @PrimaryKey
    val id: Int? = null,
    val name: String,
    val color: Int? = null
)